hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -1156,6 +1164,12 @@ bool MDMParser::checkNetStatus(NetStatus* status /*= NULL*/)
         if (!_atOk()) {
             goto failure;
         }
+        // Reformat the operator string to be numeric
+        // (allows the capture of `mcc` and `mnc`)
+        sendFormated("AT+COPS=3,2\r\n");
+        if (RESP_OK != waitFinalResp()) {
```

Comment:
This should have the proper timeout value `if (RESP_OK != waitFinalResp(nullptr, nullptr, COPS_TIMEOUT))`

Reply:
updated